hadoop-ozone/dev-support/checks/_lib.sh

```diff
@@ -40,13 +40,17 @@ _install_tool() {
     return
   fi
 
-  if [[ ! -d "${dir}" ]]; then
+  if [[ ! -d "${dir}" ]] || ! which "$bin" >& /dev/null; then
     mkdir -pv "${dir}"
     pushd "${dir}"
     if eval "${func}"; then
```

Comment:
Do you think it might be good to redirect the standard error to the output file as well?
```
if eval "${func}" 2>> "${REPORT_FILE}"; then
```

This would help us know what actually went wrong while trying to install the tool